plugins/org.jkiss.dbeaver.ext.altibase/plugin.xml

```diff
@@ -264,6 +264,11 @@
                 <folder label="%tree.administer.node.name"  id="maintenanceSettings" icon="#folder_admin" description="Maintenance/Settings">
                     <treeContribution category="connectionEditor"/>
                 </folder>
+                <folder label="%tree.system_info.node.name" icon="#folder_info" id="folderInfo" description="%tree.system_info.node.tip">
+                    <folder label="%tree.altibaseProperties.node.name" icon="#info" description="%tree.altibaseProperties.node.name">
+                        <items label="%tree.properties.node.name" path="altibaseProperties" property="altibaseProperties" icon="#info" navigable="false" virtual="true"/>
```

Comment:
The `folder` tag has a plural name, and its `items` tag has a singular name.

Also, is there a specific reason it is called `altibaseProperties` and not just `properties`?

```suggestion
                    <folder label="%tree.properties.node.name" icon="#info" description="%tree.altibaseProperties.node.name">
                        <items label="%tree.property.node.name" path="altibaseProperties" property="altibaseProperties" icon="#info" navigable="false" virtual="true"/>
```

Reply:
Thanks for your comment. 

I will change it based on the principle of using common names.